package.json

```diff
@@ -67,6 +67,7 @@
   "dependencies": {
     "@genexus/reporting-api": "~3.1.0",
     "@genexus/web-controls-library": "2.3.0",
+    "@genexus/web-standard-functions": "file:///C:/PROJECTS/genexus/web-standard-functions/dist/lib-esm",
```

Comment:
This should be a peer dependency to not force the version in this repo. Also, this should point to `@genexus/web-standard-functions`

Reply:
Great! I did this because I understood that could be necesary to modify some web-standard-function code before start working, but actually it wasn't so I will remove that. Thanks!